packages/data-point/src/ReducerEntity.js

```diff
@@ -85,11 +85,13 @@ class ReducerEntity extends Reducer {
 
       if (this.catch) {
         acc = acc.set("value", await resolveReducer(acc, this.catch));
+        if (this.outputType) {
+          await resolveReducer(acc, this.outputType);
+        }
+        return acc.value;
       }
 
-      if (this.outputType) {
-        await resolveReducer(acc, this.outputType);
-      }
+      throw error;
```

Comment:
should we have a test for this?

Reply:
Im trying to think how to replicate the issue I had, the test is being covered here 

https://github.com/ViacomInc/data-point/blob/952c62fc8b066cf95daa184c6b159f8c787c2778/packages/data-point/src/ReducerEntity.test.js#L121-L139

Reply:
get back to you in a bit

Reply:
fixed!  see test file below